src/server/pps/server/determined.go

```diff
@@ -0,0 +1,320 @@
+package server
```

Comment:
I think there probably wants to be a src/internal/determined package; that way, you don't have to include "det" or "determined" in every struct and method name.

Reply:
With all of that in mind, they appear to be using gRPC-Gateway to serve this REST API.  That means we can probably use GRPC instead of HTTP, which would be easier to deal with when things go wrong.  At the very least, gRPC-Gateway generates HTTP client stubs that we should use instead of a hand-written API.  (Now that you've written it... it doesn't matter, but if they change stuff, it might save us pain.)

Ref: https://docs.determined.ai/latest/reference/rest-api.html

Reply:
done 👍